main/RadioButton/index.tsx

```diff
@@ -56,12 +56,9 @@ const SCInputRow = styled.TouchableOpacity<IInputRowProps>`
   padding: 8px 0;
 `;
 
-const SCOuterCircle = styled.View.attrs(
-  ({ borderRadius, borderWidth }: ICircleProps) => ({
-    borderRadius,
-    borderWidth,
-  }),
-)<ICircleProps>`
+const SCOuterCircle = styled.View<ICircleProps>`
+  border-radius: ${({ borderRadius }): number => borderRadius};
```

Comment:
Is there no problem even if you don't provide the default value?

Reply:
It already has default value. Could you check the ' line:34/const DEFAULT CIRCLE' ?

Reply:
I found an additional issue. when 'InnerCircleAnim' runs, the animation value isn't placed on center.